cl/search/models.py

```diff
@@ -864,7 +865,17 @@ def save(self, update_fields=None, *args, **kwargs):
         if update_fields is not None:
             update_fields = {"slug", "docket_number_core"}.union(update_fields)
 
-        super(Docket, self).save(update_fields=update_fields, *args, **kwargs)
+        try:
+            super(Docket, self).save(
+                update_fields=update_fields, *args, **kwargs
+            )
+        except IntegrityError:
+            # Temporary patch while we solve #3359
+            # If the error is not related to `date_modified` it will raise again
+            self.date_modified = timezone.now()
+            super(Docket, self).save(
+                update_fields=update_fields, *args, **kwargs
+            )
```

Comment:
I certainly am okay with this?  @mlissner

Reply:
Yeah, I hate it, but it makes sense and we need it, so I guess let's do it.

Reply:
Thanks @mlissner - we needed to update it to deal with a failing test - not selenium related transactions.  Now that it is resolved Im going to cross my fingers.